set.mm

```diff
@@ -620078,6 +620162,20 @@ number axioms (add ~ ax-10 , ~ ax-11 , ~ ax-13 , ~ ax-nul , and remove
       ULUMUPUQBCDJQUN $.
   $}
 
+  ${
+    $d x y z A $.
+    $( A natural number that is not 1 is a successor of some other natural
+       number.  (Contributed by Steven Nguyen, 19-Aug-2023.) $)
```

Comment:
```suggestion
    $( A nonzero natural number that is not 1 is a successor of some other
       nonzero natural number.  (Contributed by Steven Nguyen, 19-Aug-2023.) $)
```
Sorry, I missed that on first reading. You can probably apply the suggestion directly here.

Reply:
I put positive integer a la nnaddcl

Reply:
Well, it's less unambiguous than "nonzero natural", but as you prefer.